requirements.pip

```diff
@@ -1,7 +1,7 @@
 Keras==2.1.6
 Pillow
 h5py
-numpy
+numpy==1.14.5
```

Comment:
Does it need to be pinned to this particular version? If so, why? 
It might cause issue if one of the other dependencies requires a newer version of numpy (tensorflow, torch etc)
What about `numpy >=1.14.5`

Reply:
This is a build level dependency not a package dependency. Fixing to the older version of numpy and building makes it compatible with future versions. The setup.py tells other packages what it depends on for runtime which can be flexible.